src/simtools/simtel/simtel_config_writer.py

```diff
@@ -216,6 +221,38 @@ def write_array_config_file(self, config_file_path, telescope_model, site_model)
                 file.write(f"# include <{tel_config_file}>\n\n")
             file.write("#endif \n\n")  # configuration files need to end with \n\n
 
+        if sim_telarray_seeds and sim_telarray_seeds.get("random_instances"):
+            self._write_random_seeds_file(sim_telarray_seeds, Path(config_file_path).parent)
```

Comment:
`Path(config_file_path).parent` should be defined as a variable since this command repeats twice.

Reply:
I've changed (although I don't think it improves the code...think complexity stays the same).